spec/models/picture_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+describe Picture, type: :model do
+  it { should have_attached_file(:image) }
+
+  it { should validate_attachment_presence(:image) }
+
+  it { should validate_attachment_content_type(:image).
```

Comment:
Style/BlockDelimiters: Avoid using `{...}` for multi-line blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)<br>Style/MultilineBlockLayout: Block body expression is on the same line as the block start.